app/assets/javascripts/tab_focus.js

```diff
@@ -48,5 +48,12 @@ function fixAnchorInURL(targetURL, csrf) {
     if (anchor.toString() === "") {
         //no anchor in url
         getAnchor(targetURL, csrf)
+    } else if (anchor.includes("&")) {
```

Comment:
there were a bug when upload file with more than 10mb the error returned back from upscan had wrong url, the anchor was in the middle when it supposed to be at the end, fixed that here